app/helpers/requests_helper.rb

```diff
@@ -45,7 +45,9 @@ def label_for_item_selector_holding(holding)
     end
   end
 
-  def queue_length_display(item, prefix: nil)
+  def queue_length_display(item, prefix: nil, title_only: false)
+    return "#{prefix}On order | No waitlist" if title_only
```

Comment:
queueTotalLength works on instance in graphql. Also you should be able to get status I think? from Instance.

Reply:
The length will just be zero when there are no items, and I will check about queueTotalLength working in this case.   I'll check about status too.

Reply:
"Status" is also returning null. I'll check with @saseestone about that.

Reply:
That instance probably has a request on it which is why it is returning 1 but if we don't need to check that is moot I think.

Yeah I don't know why I thought instance or maybe holdings had some kind of status but it doesn't look like it on the record you were looking at.

Reply:
It has a status attribute, but no value is returning, which is .. I don't know why :)

Reply:
There is a holdings summary attribute on holdingsRecord but that too is yielding nothing in this case for order status